tasks/db_deploy/orca_sql.py

```diff
@@ -549,6 +549,7 @@ def granules_table_sql() -> TextClause:
         , cumulus_granule_id    text NOT NULL
         , execution_id          text NOT NULL
         , ingest_time           timestamp with time zone NOT NULL
+        , cumulus_createdAt     timestamp with time zone NOT NULL
```

Comment:
Make lower case otherwise we will need to quote. May want to make this more descriptive like `cumulus_create_time`. At a minimum change to `cumulus_created_at`.

Reply:
changed to `cumulus_create_time`